system-tests/lib/dep-installer/index.ts

```diff
@@ -148,7 +148,7 @@ async function makeWorkspacePackagesAbsolute (pathToPkgJson: string): Promise<st
  * specified in the project's `package.json`. No-op if no `package.json` is found.
  * Will use `yarn` or `npm` based on the lockfile present.
  */
-export async function scaffoldProjectNodeModules (project: string, updateLockFile: boolean = !!process.env.UPDATE_LOCK_FILE): Promise<void> {
+export async function scaffoldProjectNodeModules (project: string, updateLockFile: boolean = !!process.env.UPDATE_LOCK_FILE, forceCopyDependencies: boolean = false): Promise<void> {
```

Comment:
The issue here was around usability/readability of the tests. By leaving the node modules in the temp directory sym linked to the fixture project, I end up getting a lot of paths that are relative from the fixture project to the temp directory rather than within the project itself.

e.g. https://github.com/cypress-io/cypress/blob/ryanm/feat/add-packherd-to-monorepo/tooling/packherd/test/entry-generator.spec.ts#L33-L36

ends up being:

```
"../../../../../../../../../tmp/cy-system-tests-node-modules/v8-snapshot/minimal/node_modules/isobject/index.cjs.js"
"../../../../../../../../../tmp/cy-system-tests-node-modules/v8-snapshot/minimal/node_modules/tmpfile/index.js"
```

instead of:

```
"node_modules/isobject/index.cjs.js"
"node_modules/tmpfile/index.js"
```

Reply:
Good call on the arguments. Will update those

Reply:
Updated here: [`c3e4634` (#23257)](https://github.com/cypress-io/cypress/pull/23257/commits/c3e46342d4b31d7bf8f5b89ccd5c512ac4e53a19)